2025-07-18 10:47:52,903 - asyncio - DEBUG - Using proactor: IocpProactor
2025-07-18 10:47:52,912 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-18 10:47:52,912 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-18 10:47:52,933 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-18 10:47:52,934 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-18 10:47:52,970 - __main__ - INFO - Telegram Manager Bot started. Send /menu to interact.
2025-07-18 10:47:52,971 - telegram.ext.ExtBot - DEBUG - Entering: send_message
2025-07-18 10:47:52,990 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-18 10:47:53,032 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000002787FB63610>
2025-07-18 10:47:53,035 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000002787FB10540> server_hostname='api.telegram.org' timeout=5.0
2025-07-18 10:47:53,056 - httpcore.connection - DEBUG - start_tls.failed exception=ConnectError(BrokenResourceError())
2025-07-18 10:47:53,059 - __main__ - ERROR - Could not send startup message to TELEGRAM_CHAT_ID 7049055170: httpx.ConnectError: 
2025-07-18 10:47:53,062 - telegram.ext.ExtBot - DEBUG - Entering: get_me
2025-07-18 10:47:53,063 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-18 10:47:53,073 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000002787FBACF10>
2025-07-18 10:47:53,073 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000002787FB10540> server_hostname='api.telegram.org' timeout=5.0
2025-07-18 10:47:53,093 - httpcore.connection - DEBUG - start_tls.failed exception=ConnectError(BrokenResourceError())
2025-07-18 10:47:53,095 - telegram.ext.Application - DEBUG - This Application is already shut down. Returning.
2025-07-18 10:47:53,098 - __main__ - ERROR - Error in polling: httpx.ConnectError: 
